pkg/operator/ceph/cluster/cluster.go

```diff
@@ -244,7 +244,7 @@ func (c *cluster) doOrchestration(rookImage string, cephVersion cephver.CephVers
 	} else {
 		// This gets triggered on CR update so let's not run that (mon/mgr/osd daemons)
 		// Start the mon pods
-		clusterInfo, err := c.mons.Start(c.Info, rookImage, cephVersion, *c.Spec, c.isUpgrade)
+		clusterInfo, err := c.mons.Start(c.Info, rookImage, cephVersion, *c.Spec, c.isUpgrade, c.Spec.SkipUpgradeChecks)
```

Comment:
since we're already passing `c.Spec`, do we still need to pass `c.Spec.SkipUpgradeChecks` here?